### ISSUE TITLE
Fix warnings for newer PyTorch versions

### ISSUE THREAD

**Issue by themightyoarfish** [Peltarion/ai_ikkuna#30]
This includes correct use of `softmax()` which now expects an explicit dimension, and tensor copies in the Exporter.
